Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM fedora:latest
 # e2fsprogs -- docker @ F20 wants it
```

Comment:
Since we're going with `dnf` which was adopted on Fedora 22 can we remove the e2fsprogs comment as well? It seems to suggest it's only needed for Fedora 20.